pages/contact.tsx

```diff
@@ -0,0 +1,129 @@
+// pages/contact.js
+import React, {useState} from 'react';
+import Navbar from "@/components/navbar/NavBar"; // Assuming you have a Navbar component
+import Image from "next/image";
+import Logo from "../public/logo-svg.svg"; // Update the path as necessary
+import ModalWindow from "@/components/DemoPage/ModalWindow";
+
+const ContactPage = () => {
+    const [fullName, setFullName] = useState('');
+    const [email, setEmail] = useState('');
+    const [message, setMessage] = useState('');
+    const [errors, setErrors] = useState({fullName: '', email: ''});
+    const [isModalVisible, setIsModalVisible] = useState(false);
+
+    // rest of your code...
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+        let newErrors = { fullName: '', email: '' };
+        let isValid = true;
+
+        // Validate full name
+        if (!fullName) {
+            newErrors.fullName = 'Full name is required';
+            isValid = false;
+        }
+
+        // Validate email format
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+        if (!emailRegex.test(email)) {
+            newErrors.email = 'Invalid email format';
+            isValid = false;
+        }
+
+        if (isValid) {
+            try {
+                // Call the API to send the email
+                const response = await fetch('/api/send-email', {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify({ fullName, email, message }),
+                });
+                const data = await response.json();
+                if (data.success) {
+                    console.log('Email sent successfully');
+                    setIsModalVisible(true);
+                    // Reset form fields
+                    setFullName('');
+                    setEmail('');
+                    setMessage('');
+                } else {
+                    console.log('Email sending failed');
+                    // Handle failure
+                }
+            } catch (error) {
+                console.error('Failed to send email', error);
+                // Handle error
+            }
+        } else {
+            setErrors(newErrors);
+        }
+    };
+
+    return (
+            <div
+                className="min-h-screen bg-gradient-to-b from-linear-gradient-start to-linear-gradient-end flex flex-col">
+                <Navbar/>
+                <ModalWindow isVisible={isModalVisible} onClose={() => setIsModalVisible(false)} />
+                <div className="flex flex-grow flex-col items-center justify-center pt-16 pb-6">
+                    <Image src={Logo} alt="Logo" width={50} height={50}/>
+                    <div className="mt-10 w-full max-w-md p-8 space-y-6 bg-white rounded-lg shadow-md">
```

Comment:
```suggestion
                    <div className="mt-10 w-full max-w-md p-8 space-y-6 bg-chatgenieWhite rounded-lg shadow-md">
```
It will be easier to create different themes when custom colors used

Reply:
@KamronBegmatov mambilani qimapsizku hali oka